stdpopsim/genomes.py

```diff
@@ -410,7 +463,16 @@ def species_contig(
     def length(self):
         return self.recombination_map.sequence_length
 
-    def dfe_breakpoints(self):
+    @property
+    def original_coordinates(self):
+        """
+        If the contig represents a chromosomal segment, this is gives the name
+        of the parent chromosome and the left and right coordinates of the
+        contig on this chromosome.
+        """
+        return stdpopsim.utils.parse_region_string(self.origin)
```

Comment:
I think it would be better to store the original coordinates as Contig fields and produce the `origin` property from them, instead of the storing and parsing the origin string.